cypress/integration/navigation-test.js

```diff
@@ -33,4 +33,10 @@ describe('User navigation capabilities', () => {
       expect(loc.pathname).to.eq('/')
     })
   })
+
+  it('Should display a bad path message to all invalid paths', () => {
```

Comment:
Let's meet with BE to determine if all numbers or certain searches are not worth sending to them